### PR TITLE
 I  have changed the Theme of Counter app and made it similar to the …

### PR DESCRIPTION
**Pull request by Nagsenwaghmare**
…calculator app please have a look at it and if there are any suggestions on which I can work more let me know .
Thank you!

### CODE REVIEW

File: public/counter.html [hunk 1]
```diff
@@ -1,51 +1,24 @@
...
-        <div>
-            <button id="decrease">Decrease Count</button>
-            <button id="reset">Reset</button>
-            <button id="increase">Increase Count</button>
-        </div>
+      <h1>Counter</h1>
+      <div>
+        <span id="count">0</span>
+      </div>
+      <div>
```
Comment (Vishal-raj-1) [inline public/counter.html:15]: What is the need of giving different class to this button ? You can use "id" for that purpose. And always try to give meaning full name.

**Review by Vishal-raj-1 — COMMENTED** [review]
Code looks good to me but always send the video clip OR screenshots so that I can review the UI.

File: public/counter.html (hunk 1 above)
Comment (Nagsenwaghmare) [reply public/counter.html:15]: Thanks for the suggestion I will work on it and will commit those changes shortly with Screenshot .

Comment (Nagsenwaghmare) [reply public/counter.html:15]: Counter - Google Chrome 08-02-2021 21_01_09

Comment (Nagsenwaghmare) [reply public/counter.html:15]: This is the UI as same as You have done with calculator

**Review by Vishal-raj-1 — APPROVED** [review]
Thanks for you contribution !! Looking good to me.

**Pull request merged**